tldraw/tldraw
# refactor: improve stroke appearance and update default size

This PR improves the appearance of strokes for all shapes and makes the default size small (resolves #508).

The math is still quite complex and varies between shapes more than I would like.

### Change type


### Test plan

1. Create various shapes (arrow, draw, ellipse, rectangle) and verify stroke appearance.
2. Verify that the default size for new shapes is small.


### Release notes

- Improved the appearance of strokes across all shapes and updated the default size to small.
